configs/h_deformable_detr/deformable_detr_r50_trick_1x_coco.py

```diff
@@ -0,0 +1,184 @@
+_base_ = [
```

Comment:
Would you mind making the original two-stage deformable-detr as _base_ then we can notice all modifications which indicate the tricks